include/openenclave/host.h

```diff
@@ -127,6 +131,9 @@ typedef struct _oe_enclave_setting
     union {
         const oe_enclave_setting_context_switchless_t*
             context_switchless_setting;
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+        oe_eeid_t* eeid;
```

Comment:
can this be const like line 132 is?

Reply:
Not at the moment. It's used as storage for the various bits that need to be saved during enclave creation (and in the non-plugin version I abuse it as an output parameter). I think for the plugin-only setting I can create an internal copy though.